packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -369,14 +369,22 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     }
 
     return (
-      <div ref={this._root} {...divProps} role={role === undefined ? 'list' : role} className={css('ms-List', className)}>
+      <div ref={this._root} {...divProps} role={this._getRole(role, pageElements)} className={css('ms-List', className)}>
         <div ref={this._surface} className={css('ms-List-surface')} role="presentation">
           {pageElements}
         </div>
       </div>
     );
   }
 
+  private _getRole(role: string | undefined, pageElements: JSX.Element[]): string | undefined {
```

Comment:
I don't think this logic is complicated enough to warrant its own function. Grab me and we can consolidate.